client/components/navbar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap-icons/font/bootstrap-icons.css';
```

Comment:
Same for the Bootstrap Icons, just import them in the `App` component, then you don't need to import them anywhere else.